### ISSUE TITLE
Review SputnikVM code base

### ISSUE THREAD

**Issue by artob** [near/near-evm#57]
As discussed on our 2021-02-01 call.

https://github.com/rust-blockchain/evm

**Comment by artob** [comment]
Capturing here some of the other trade-offs we discussed yesterday with @ilblackdragon, @djsatok, and @frankbraun:

- Only 5,000 lines of Rust code, facilitating a full understanding of the whole implementation.
- Requires only a minimal set of dependencies compared to OpenEthereum.
- Up-to-date dependencies. For example, importantly, [primitive-types](https://crates.io/crates/primitive-types) is up-to-date.
- On the downside, the project has hardly any test coverage.
- On the downside, full Berlin HF support is not evident in the upstream code base as yet.

The bottom line is that the pros likely outweigh the cons, and we've decided to evaluate using SputnikVM going forward.

**Comment by artob** [comment]
Capturing here some of the other trade-offs we discussed yesterday with @ilblackdragon, @djsatok, and @frankbraun:

- Only 5,000 lines of Rust code, facilitating a full understanding of the whole implementation.
- Requires only a minimal set of dependencies compared to OpenEthereum.
- Up-to-date dependencies. For example, importantly, [primitive-types](https://crates.io/crates/primitive-types) is up-to-date.
- On the downside, the project has hardly any test coverage.
- On the downside, full Berlin HF support is not evident in the upstream code base as yet.

The bottom line is that the pros likely outweigh the cons, and we've decided to evaluate using SputnikVM going forward.
